demos/inpage/src/EthereumApi.tsx

```diff
@@ -270,29 +374,38 @@ export default class EthereumApi {
         initCode,
         callData,
         callGasLimit: actions.map((a) => BigInt(a.gas)).reduce((a, b) => a + b),
```

Comment:
Yeah it makes sense you can set to zero. We would need to measure this anyway because we have a simulated bundler. Additionally, you're supposed to be able to set a gas limit manually in the API, and we need to honor that when it is provided instead of doing our own calculation.